src/Models/ShallowWaterModels/ShallowWaterModels.jl

```diff
@@ -16,6 +18,12 @@ include("set_shallow_water_model.jl")
 ##### Time-stepping ShallowWaterModels
 #####
 
-# Coming soon!
+"""
+    fields(model::ShallowWaterModel)
+Returns a flattened `NamedTuple` of the fields in `model.solution`.
+"""
+fields(model::ShallowWaterModel) = merge(model.solution, model.tracers)
+
+include("calculate_shallow_water_tendencies.jl")
```

Comment:
This PR edits two files: `calculate_shallow_water_tendencies.jl` and `calculate_tendencies.jl`. But `calculate_tendencies.jl` is not included here. What is the intended behavior?